hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/CryptoCreateHandler.java

```diff
@@ -49,10 +49,12 @@ public CryptoCreateHandler() {
     public void preHandle(@NonNull final PreHandleContext context) {
         requireNonNull(context);
         final var op = context.getTxn().cryptoCreateAccountOrThrow();
-        final var key = asHederaKey(op.key());
-        final var receiverSigReq = op.receiverSigRequired();
-        if (receiverSigReq && key.isPresent()) {
-            context.addToReqNonPayerKeys(key.get());
+        if (op.hasKey()) {
+            final var key = asHederaKey(op.key());
```

Comment:
I'm seeing a "null" warning on this line.

Reply:
Yes, used `keyOrThrow()`. Thank you!